auth-web/src/components/auth/common/LearnMoreButton.vue

```diff
@@ -21,4 +23,13 @@ export default class LearnMoreButton extends Vue {}
     font-weight: bold;
     width: 160px;
   }
+
+  .btn-learn-more:hover {
+    background-color: $BCgovBlue5;
+    opacity: .8;
```

Comment:
Again with the opacity. Why is this different from the other UIs?

Reply:
This is a mistake! Doubled up. Though like before, should we go local on these btn:hovers opposed to global. I'm 100% good with either. Maybe local would be safer..

Reply:
Go safer to preempt full regression testing.

Reply:
Updated in next commit. Removed the duplication!